### ISSUE TITLE
Shooting bracketed sequences may fail

### ISSUE THREAD

**Issue by houz** [darktable-org/darktable#4755]
When shooting bracketed sequences in tethering view dt triggers the captures as fast as it can. However, at least some cameras don't like that and report that the camera is still busy:

```
9.636410 [camera_control] executing set camera config job shutterspeed=0.3333s
9.762542 [camera_control] executing remote camera capture job
[import_session] Using filename 20200418_0007.jpg.
14.003063 [camera_control] executing set camera config job shutterspeed=0.2500s
14.028634 [camera_control] gphoto2 error: The property 'Shutter Speed' / 0x500d was not set (0x2019: PTP Device Busy)
14.028742 [camera_control] executing remote camera capture job
14.036174 [camera_control] disabling tether mode
14.036189 [camera_control] camera control un-locked for camera 0xaf009168
14.036215 [camera_control] unregistering listener 0x25fe518
14.036246 [camera_control] unregistering listener 0x2b36cb8
14.036317 [camera_control] disabling tether mode
14.036322 [camera_control] camera control un-locked for camera (nil)
14.036329 [camera_control] unregistering listener 0x25fe518
```

The relevant code is in https://github.com/darktable-org/darktable/blob/master/src/control/jobs/camera_jobs.c#L131-L161. One quick fix is to use `params->delay` inside the for loop. However, that requires the user to figure out how long he needs to wait between shots and is error prone as many people will not set a delay at all. So a better solution is to make the shots blocking.

**Comment by johnny-bit** [comment]
I think I have fixed that in #7126 somewhat - delay is used... But I agree that blocking shot would be better alternative

**Comment by johnny-bit** [comment]
Still needed for nikons.
